bundle/config/root.go

```diff
@@ -461,6 +466,37 @@ func validateVariableOverrides(root, target dyn.Value) (err error) {
 	return nil
 }
 
+// normalizeVariableLookupOverrides makes sure that variables which are overriden in targets with lookup
+// do not have any default value set because otherwise the lookup will not be performed.
```

Comment:
Could you also add why lookup will not be performed? That is default value will be directly used. It's kind of implied but making it clear would be better.

Reply:
What happens if I set both default and lookup in root config? What about both in a target override? Do we need to emit a warning in those cases?

Reply:
In this case lookup would be ignored and we log a debug message for this